### PR TITLE
feat: make format compatible with "PER Coding Style 2.0"

### PR DESCRIPTION
**Pull request by ferchoz**
This change allows to use this plugin along with "PER Coding Style 2.0"
formating :
`readonly final class ClassName {}`
to
`final readonly class ClassName {}`

doc:
https://www.php-fig.org/per/coding-style/#46-modifier-keywords

### CODE REVIEW

**Comment by ferchoz** [comment]
Hi @czosel  can you take a look a this PR?
Thanks!

**Comment by PedroTroller** [comment]
I want to make this change, I look at the PRs and I realise that I'm 5 days late... Thanks @ferchoz ;)
You can probably rename your PR to specify that it's not just to make it compatible with php-cs-fixer but above all with the [php-fig.org PER standard](https://www.php-fig.org/per/coding-style/#46-modifier-keywords).

File: src/printer.js [hunk 1]
```diff
@@ -1204,14 +1204,14 @@ function printClass(path, options, print) {
...
-
   if (node.isFinal) {
     declaration.push("final ");
   }
 
+  if (node.isReadonly) {
+    declaration.push("readonly ");
+  }
+
   if (node.isAbstract) {
```
Comment (czosel) [inline src/printer.js:1215]: Shouldn't this also be moved up up before `isReadonly`? PER says

> 1. Inheritance modifier: abstract or final
> 2. Visibility modifier: public, protected, or private
> 3. Scope modifier: static
> 4. Mutation modifier: readonly

**Review by czosel — CHANGES_REQUESTED** [review]
Hi @ferchoz, thanks for contributing and sorry about the delay! The idea looks good to me, just a minor point for completeness.

File: src/printer.js (hunk 1 above)
Comment (ferchoz) [reply src/printer.js:1215]: Awesome! let me improve it

**Comment by ferchoz** [comment]
> Hi @ferchoz, thanks for contributing and sorry about the delay! The idea looks good to me, just a minor point for completeness.

Improved!

**Comment by ferchoz** [comment]
@czosel can you take a look at it? thanks!!

**Review by czosel — APPROVED** [review]
Looks great! Thanks for contributing @ferchoz

**Comment by czosel** [comment]
Released in `v0.22.0` :tada:

**Pull request merged**